README.md

```diff
@@ -92,29 +92,15 @@ must be escaped manually.
 Configuration
 -------------
 
-### PDF viewer
+### Option A: PDF viewer and TeX engine and Synctex support
+At the moment of writing the version of 'vim-latex-live-preview' contained in this fork is needed.
+For the setup details see [my gist](https://gist.github.com/andreas-wachtel/1025a7d2c246af267da2b84234f57d3f).
+I still intent to put a screen-cast.
 
-By default, you need to have [evince][] or [okular][] installed as pdf viewers.
-But you can specify your own viewer by setting `g:livepreview_previewer`
-option in your `.vimrc`:
```

Comment:
Could you clean up README? You can create a new branch in your repo so it won't conflict with the description of your repo.

Reply:
sure (give me some time please).